bears/configfiles/DockerfileLintBear.py

```diff
@@ -22,6 +22,8 @@ class DockerfileLintBear:
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
     CAN_DETECT = {'Syntax', 'Smell'}
+    SEE_MORE = ('https://github.com/projectatomic'
+                + '/dockerfile_lint#dockerfile-lint')
```

Comment:
can you have this on a single line?

Reply:
I had tried that first but that would make the line of 81 characters which will not follow PEP8 styles
I also tried using
# Ignore PycodestyleBear (E501)
    SEE_MORE = 'https://github.com/projectatomic/dockerfile_lint#dockerfile-lint'
on a different branch on my fork...and tested it with these bots...
Its giving the same errors :disappointed:

Reply:
@anctartica, then bot will catch warning of line exceeding 72 characters 😅

Reply:
Please use curly bracket @harshhx17. And I think https://github.com/projectatomic/dockerfile_lint this much link is enough.